### PR TITLE
Add command line argument to specify target

### PR DESCRIPTION
**Pull request by L2jLiga**
issue #36 

*Usage*:
```
npx downlevel-dts . ts3.8 --to=3.8
```

### CODE REVIEW

**Review by sandersn — CHANGES_REQUESTED** [review]
Need to use the semver package instead of number parsing.

File: baselines/ts4/test.d.ts [hunk 1]
```diff
@@ -0,0 +1,51 @@
+/// <reference path="./src/test.d.ts" />
```
[the remark this answers is not in the document]
Comment (L2jLiga) [reply baselines/ts4/test.d.ts:1]: Right

File: index.js [hunk 2]
```diff
@@ -39,20 +40,33 @@ module.exports.main = main;
 if (!(/** @type {*} */ (module.parent))) {
   const src = process.argv[2];
   const target = process.argv[3];
-  main(src, target);
+  const to = process.argv.find(arg => arg.startsWith("--to"));
+  let targetVersion = 3.4;
+  if (to) {
+    const targetVersionArg = parseFloat(to.split("=")[1]);
```
[the remark this answers is not in the document]
Comment (L2jLiga) [reply index.js]: Sure, I updated PR and now it uses semver

**Review by sandersn — APPROVED** [review]

**Comment by sandersn** [comment]
Having the test output in the `test` folder was causing multiple test runs to read output as input, so I moved to the baselines/local and baselines/reference structure that TS itself uses.

**Pull request merged**